les/handler_test.go

```diff
@@ -452,7 +394,7 @@ func TestGetBloombitsProofs(t *testing.T) {
 			cs, _, _ := cIndexer.Sections()
 			bs, _, _ := bIndexer.Sections()
 			bts, _, _ := btIndexer.Sections()
-			if cs >= 8 && bs >= 8 && bts >= 1 {
+			if cs >= 1 && bs >= 1 && bts >= 1 {
```

Comment:
the `bs` should >= 8 since we generate 8 sections `bloom data` and 1 `bloom trie root`

Reply:
You're right. Actually there is no reason to check bs here since btIndexer can only process its sections after bIndexer is ready. Checking cs is also unnecessary here because we are not using it.